python/spacewalk/spacewalk-backend.spec

```diff
@@ -41,7 +41,7 @@
 %global apache_user wwwrun
 %global apache_group www
 %global apache_pkg apache2
-%global documentroot /usr/share/susemanager/www/htdocs
+%global documentroot /srv/www/htdocs
```

Comment:
The idea behind moving this away from `/srv` is to have as few RPM-managed files in `/srv` to simplify the container mounts. I would rather keep the htdocs in `/usr/share/susemanager/www/htdocs` for that reason